docs/semgrep-sc/sc-glossary.md

```diff
@@ -51,12 +51,12 @@ Continuous integration scans with Semgrep Supply Chain rules block any merge or
 
 See also [Reachability](#reachability).
 
-
 ## Reachability
 
 Reachability refers to whether or not a vulnerable piece of code from a dependency is used in the codebase itself. In Semgrep Supply Chain, both a dependency's vulnerable version and code pattern must match for a vulnerability to be considered reachable.
 
-Semgrep Supply Chain performs reachability analysis to assist in triaging vulnerabilities. For more information, see Getting started with Semgrep Supply Chain.
+See [Semgrep Supply Chain Overview documentation](/docs/semgrep-sc/semgrep-supply-chain-overview) to learn how Semgrep leverages its code-scanning and rule syntax capabilities to provide high-signal rules that determine a finding's reachability. This assists security engineers in remediation and triage processes.
```

Comment:
@s-santillan just a small note - for absolute internal links (which work the best when we are changing the structure of the docs, so a good choice), do not include the `/docs/` in the beginning. The root for links in `/` is considered as `/docs/` already. It causes some issues which you can see if you run `yarn build && yarn serve`. 🙂 

```suggestion
See [Semgrep Supply Chain Overview documentation](/semgrep-sc/semgrep-supply-chain-overview) to learn how Semgrep leverages its code-scanning and rule syntax capabilities to provide high-signal rules that determine a finding's reachability. This assists security engineers in remediation and triage processes.
```

Reply:
Maybe pull from the main branch, and check if these links are still present, and change the remaining ones, please.